Livecheckables/xmlto.rb

```diff
@@ -0,0 +1,6 @@
+class Xmlto
+  livecheck do
+    url "https://releases.pagure.org/xmlto/?C=M&O=D"
+    regex(/href=.+xmlto-v?(\d+(?:\.\d+)+)\.t/)
```

Comment:
```suggestion
    regex(/href=.*?xmlto-v?(\d+(?:\.\d+)+)\.t/i)
```

There are a couple updates in our regex standards here to make you aware of:

I've settled on using `href=.*?` for the moment (instead of `href=.+?`), since I once encountered a page that didn't use any quotation marks around attributes (the horror!). That said, regardless of whether we use `.*` or `.+`, we want to make sure to make it a non-greedy match with `?` like `href=.*?`. If we don't, the greedy match can easily escape the bounds of the `href` attribute and cause trouble for us.

I've also decided to use case insensitive regexes unless we specifically need case sensitivity for a specific check. This is the `i` at the end of the regex literal here (i.e., after the closing `/` or `}`). Basically, this will help to reduce the chances of a check breaking simply due to a change in text case.

Reply:
Thank you (as always) for the help and explanation @samford.